src/main/webapp/WEB-INF/views/drivers/all.jsp

```diff
@@ -8,10 +8,12 @@
     <title>All drivers</title>
 </head>
 <body>
+<%@include file="/WEB-INF/views/header.jsp" %>
 <h1 class="table_dark">All drivers:</h1>
 <table border="1" class="table_dark">
     <tr>
         <th>ID</th>
+        <th>Login</th>
```

Comment:
if you started adding login and pass to tables, would be correctly to add it everywhere